### PR TITLE
Replace batch scripts with powershell ones, now they just proxy to ps…

### PR DESCRIPTION
**Pull request by ctomc**
…1 version of scripts

this is optional PR, that should be merged after the new scripts are validated by qe

### CODE REVIEW

File: core-feature-pack/src/main/resources/content/bin/jboss-cli.ps1 [hunk 1]
```diff
@@ -16,6 +16,7 @@ Function Set-Env {
   Set-Content -Path env:$key -Value $value
 }
 
+<<<<<<< HEAD
```
Comment (Ladicek) [inline core-feature-pack/src/main/resources/content/bin/jboss-cli.ps1]: Is this some special powershell feature? :-)

Comment (ctomc) [inline core-feature-pack/src/main/resources/content/bin/jboss-cli.ps1]: ups, how did that get there :) there was some merge conflict when i was splitting the PR that had two commits that added/fixed powershell scripts and one that is now essentially this.

Comment (ctomc) [inline core-feature-pack/src/main/resources/content/bin/jboss-cli.ps1]: fixed

**Comment by jamezp** [comment]
My fear here is the PowerShell scripts have not been tested and these have. I'm a bit on the paranoid side though especially this close to final :)

**Comment by ctomc** [comment]
That is why the separate PR, which can be merged in later time, preferably after 10.Final that has new ps1. scripts in, is released.

**Comment by ctomc** [comment]
closing for now, will open when time is right

**Pull request closed without merging**